### PR TITLE
Support more printf formats

### PR DESCRIPTION
**Pull request by kant2002**

### CODE REVIEW

File: Cesium.IntegrationTests/stdlib/io/fprintf.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    printf("Characters:\t%c %%\n", 'A');
+
+    printf("Integers:\n");
+    printf("\tDecimal:\t%i %d %.6i %i %.0i %+i %i\n",
+        1, 2, 3, 0, 0, 4, -4);
+    printf("\tHexadecimal:\t%x %x %X %#x\n", 5, 10, 10, 6);
+    printf("\tOctal:\t\t%o %#o %#o\n", 10, 10, 4);
+
+    printf("Floating point:\n");
+    printf("\tRounding:\t%f %.0f %.32f\n", 1.5, 1.5, 1.3);
```
Comment (kant2002) [inline Cesium.IntegrationTests/stdlib/io/fprintf.c:26]: I would comment out %f.32 format for now.

Comment (ForNeVeR) [reply Cesium.IntegrationTests/stdlib/io/fprintf.c:26]: Yeah, let's comment it out for now.

**Review by ForNeVeR — APPROVED** [review]
Other than a failing test, I see no issues with this code.

**Review by ForNeVeR — APPROVED** [review]
Thanks!

**Pull request merged**